tests/git.analysis.spec.ts

```diff
@@ -40,5 +40,17 @@ describe('Functional test of analysis', () => {
     );
     expect(Object.keys(bundle.analysisResults.files).length).toEqual(1);
     expect(Object.keys(bundle.analysisResults.suggestions).length).toEqual(1);
+    
+  });
+  it('analyze remote git with oid and return sarif', async () => {
+    const bundle = await analyzeGit(
+      baseURL,
+      sessionToken,
+      false,
+      1,
+      'git@github.com:DeepCodeAI/cli.git@320d98a6896f5376efe6cefefb6e70b46b97d566',
+      true,
+    );
+    expect(!!bundle.sarifResults).toEqual(true);
```

Comment:
You would probably want to have some format specification here so that if we break something in the future the tests will tell us that the output format is wrong.

Reply:
As discussed Type script already infers the output type and will not compile if we break something.